### PR TITLE
feature/DAP-9

### PR DESCRIPTION
**Pull request by Viliusps**

### CODE REVIEW

File: api/dating_app/src/main/java/com/psap/dating_app/model/User.java [hunk 1]
```diff
@@ -93,28 +92,24 @@ public class User {
...
 
-    @NotBlank(message = "Star sign is mandatory")
     @Column(name = "star_sign", nullable = false)
     private String starSign;
 
-    @NotBlank(message = "Personality type is mandatory")
     @Column(name = "personality_type", nullable = false)
     private String personalityType;
 
-    @NotBlank(message = "Love language is mandatory")
```
Comment (simsve3) [inline api/dating_app/src/main/java/com/psap/dating_app/model/User.java:113]: Why delete these annotations?

File: api/dating_app/src/main/java/com/psap/dating_app/service/UserService.java [hunk 2]
```diff
@@ -20,14 +22,40 @@ public User getUserById(long id) {
...
     public boolean existsUser(long id) {
         return userRepository.existsById(id);
     }
 
+    public User login(LoginRequest request) throws IllegalAccessException {
+
+        if(userRepository.existsByEmail(request.getEmail()))
+        {
+            User userFromDb = userRepository.findByEmail(request.getEmail());
+            System.out.println(userFromDb);
```
Comment (simsve3) [inline api/dating_app/src/main/java/com/psap/dating_app/service/UserService.java]: Is this print necessary for code to work?

File: api/dating_app/src/main/resources/db/migration/V2__users_table.sql [hunk 3]
```diff
@@ -6,21 +6,21 @@ CREATE TABLE users
...
     points              INT,
     block_end           TIMESTAMPTZ,
     blocked             BOOLEAN,
     star_sign           VARCHAR(255),
     personality_type    VARCHAR(255),
     love_language       VARCHAR(255),
-    match_purpose       purposes,
+    match_purpose       VARCHAR(255),
   	CONSTRAINT users_pk PRIMARY KEY (id)
 );
```
Comment (simsve3) [inline api/dating_app/src/main/resources/db/migration/V2__users_table.sql:26]: - Add two or three user login information to README (two of them should match).
- Duplicate emails causes user id to be undefined in frontend

**Review by simsve3 — CHANGES_REQUESTED** [review]
Everything else LGTM

File: api/dating_app/src/main/java/com/psap/dating_app/model/User.java (hunk 1 above)
Comment (Viliusps) [reply api/dating_app/src/main/java/com/psap/dating_app/model/User.java:113]: nes kol neturim to kazkokio testo kur sita nustato palikau kad blank gali but, kai implementuosim nustatyma sito padarysim not blank jei jis vps butinas bus

**Review by simsve3 — APPROVED** [review]

**Pull request merged**